### ISSUE TITLE
Pretrained models seem incompleted?

### ISSUE THREAD

**Issue by Lansie0909** [yan-hao-tian/ConTNet#7]
Thanks for your great job! I tried to load your pretrained model of ImageNet to train ConTNet on my own dataset, but there is only a tensor in your two pretrained models, I can't load best acc or optimizer and scheduler. And for model loading, I modify your code to "strict=False" to load model, otherwise every layer of the model can't be loaded.  In the end, my training accuracy is very low. I think this is related to the incorrect loading of the pretrained model. Can you help me solve this problem by publishing a complete pretrained model or point out my errors in loading？Thanks a lot!
![0ON852 @KD) 4 )9ITW%ZT9](https://user-images.githubusercontent.com/58927039/148630934-cc6f6b0f-82a3-43be-a03c-8a8f221053a0.png)
